packages/permission-controller/ARCHITECTURE.md

```diff
@@ -1,4 +1,4 @@
-# PermissionController
+# Architecture
```

Comment:
Nit: It would seem appropriate to incorporate this into the README or at least link to it from there.

Reply:
Done here: [b6d186d](https://github.com/MetaMask/controllers/pull/831/commits/b6d186d05b632083b311a04ed895f483a73d4942). I guess we could move it to README but it seemed a bit large. I admit "Architecture" isn't the best name, however. I could go either way on this.